doc/examples/features_detection/plot_scattering.py

```diff
@@ -0,0 +1,143 @@
+"""
+===================
+Scattering Features
+===================
+
+This example shows how to use the scattering transform on images.
+It explains how to access easily different scattering
+coefficients stored in the output vector. For more information on
+how to use scattering coefficients for image
+classification, please check the example *Scattering features
+for supervised learning*.
+We call the scattering features (or coefficients) the output of the
+scattering transform applied to an image.
+These features have been shown to outperform any other 'non-learned'
+image representations for image
+classification tasks [2]_. Structurally, they are very similar to
+deep learning representations, with fixed filters
+(here Morlet filters).
+Since the scattering transform is based on the
+Discrete Wavelet transform (DWT), it is stable to deformations.
+It is also invariant to small translations. For more details on
+its mathematical properties, see [1]_.
+
+**Definition of the scattering coefficients**
+
+Given an image :math:`x`, the scattering transform computes
+(generally) three layers of cascaded convolutions
+and non-linear operators:
+
+-*Zero-order coefficients:*
+:math:`Sx[0] = x * \phi`
+
+-*First-order coefficients:*
+:math:`Sx[1][(i,l)] = | x * \psi_{(i,l)} | * \phi`
+
+-*Second-order coefficients:* :math:`Sx[2][(i,l)][(j,l_2)]=|| x * \psi_{(i,l)}|*\psi_{(j,l_2)}|*\phi`,
+ where :math:`\psi` is a band-pass filter, :math:`\phi` is a low-pass filter
+ (normally a Gaussian), operator :math:`*` is a 2D convolution,
+ and :math:`|\cdot|` is
```

Comment:
sorry for this comment, but these lines are now super short. The easiest to make your code pep8 compatible wrt the 80 column criterion is to use a function in the editor that will try to adjust every line of the doc to as close to 80 as possible.